### PR TITLE
Balanced_braces

### PR DESCRIPTION
**Pull request by rbhattad**
program to check whether braces are balanced are not using stack

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [ ] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [ ] Relevant documentation/comments is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.

Notes: 

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C-Plus-Plus/pull/1431"></a>

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Code is not up to the [repository standards](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md).
Please read them **carefully** and **follow them**.

**Comment by rbhattad** [comment]
this is my first contribution  please give more info. for enhancement and documentation i have added proper comments

**Review by Panquesito7 — COMMENTED** [review]
See #962 for how to make your code refined and well-documented.
Or, see the [typical structure](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#typical-structure-of-a-program) of a program for more information. 🙂

File: data_structures/Balanced_Bracket [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	char string[50];   //input String
+	cin.getline(string,48);
+	if(!check(string))  //check
+	cout<<"correctly paranthesis";
+	else
+	cout<<"Incorrectly paranthesis";
+	
+//	checkfirst(string);
+//	checklast(string);
+//	checknumber();
```
Comment (AnupKumarPanwar) [inline data_structures/Balanced_Bracket:90]: Are these comments here for some purpose?

**Pull request closed without merging**